libraries/botbuilder-ai-orchestrator/src/orchestratorAdaptiveRecognizer.ts

```diff
@@ -287,4 +379,52 @@ export class OrchestratorAdaptiveRecognizer
             this._resolver = OrchestratorAdaptiveRecognizer.orchestrator.createLabelResolver(snapshot);
         }
     }
+
+    private async tryScoreEntities(text: string, recognizerResult: RecognizerResult) {
+        if (!this.scoreEntities) {
+            return;
+        }
+
+        const results = await this._resolver.score(text, LabelType.Entity);
+        if (!results) {
+            throw new Error(`Failed scoring entities for: ${text}`);
+        }
+
+        // Add full entity recognition result as a 'result' property
+        recognizerResult[this.entityProperty] = results;
+        if (results.length) {
+            recognizerResult.entities ??= {};
+
+            results.forEach((result: Result) => {
+                const entityType = result.label.name;
+
+                // add value
+                const values = recognizerResult.entities[entityType] ?? [];
+                recognizerResult.entities[entityType] = values;
+
+                const span = result.label.span;
+                const entityText = text.substr(span.offset, span.length);
+                values.push({
+                    type: entityType,
+                    score: result.score,
+                    text: entityText,
+                    start: span.offset,
+                    end: span.offset + span.length,
+                });
+
+                // get/create $instance
+                const instanceRoot = recognizerResult.entities['$instance'] ?? {};
+
+                // add instanceData
+                const instanceData = instanceRoot[entityType] ?? [];
```

Comment:
Will the creation take effect on `recognizerResult.entities['$instance']`?  Thanks.

Reply:
You may need to do
```
recognizerResult.entities['$instance'] ?? = {};
const instanceRoot = recognizerResult.entities['$instance'];
```
and the same for `instanceData`.